app/javascript/react/components/main.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './bands/App'
+ReactDOM.render(
+  <App />,
+  document.getElementById('app')
+)
```

Comment:
an app/javascript/packs/application.js file already exists for this application by default and handles the initial render of a component. I dont believe we need to create a new file for this